toast/toast.js

```diff
@@ -0,0 +1,13 @@
+const baselineBtn = document.querySelector(".baseline-btn");
+const baselineToast = document.querySelector(".baseline-toast");
+const toastBtn = document.querySelector(".toast-btn");
+
+const hideToast = () => (baselineToast.style.visibility = "hidden");
+const showToast = () => {
+  baselineToast.style.visibility = "visible";
+  setTimeout(hideToast, 5000);
```

Comment:
I feel 5 seconds are too much. You can make it 3 seconds.

Reply:
okay noted !